README: add a short statement of how Tailscale uses golink (#240)
tailscale/golink: README.md

![Screenshot of golink home screen](screenshot.png)

## How Tailscale uses golink

> We use golink inside Tailscale every day by every part of the company.

Reviewer: Take it or leave it nit: maybe
```suggestion
golink is used inside Tailscale every day by every part of the company.
```
The current wording feels a little off to me.

Author: That was actually what I had originally, but I couldn't decide on whether "golink" should be capitalized at the beginning of a sentence, so I changed the wording so I didn't have to decide 😛

Reviewer 2: FWIW I kind of preferred the active voice, but YMMV.

Reviewer: Hmm maybe something like "We use golink internally every day across every part of the company" if we want active voice?

It is easily one of the most used services on our corporate tailnet.
We even had to change our new hire onboarding to have employees join our tailnet sooner,
since much of the rest of our onboarding involved visiting various go links.

Our production environment is pretty plain; we just run it on a pretty vanilla EC2 VM.

> We [backup](#backups) all of our go links once a week from a GitHub Action that stores the snapshot in our internal git monorepo.

Reviewer 3: "back up" is the phrasal verb. "backup" is a noun. 🤓

> That repo has a wrapper script around `golink -resolve-from-backup` so that anyone with a local copy of the repo can always resolve go links offline.

Reviewer 3: if you fix the above, add a double hyphen here? we try to do that elsewhere in Tailscale docs, as it looks less weird to people than Go's style.
